CMakeLists.txt

```diff
@@ -17,29 +17,28 @@ set(CMAKE_CXX_STANDARD 11)
 # Third Party Dependencies
 find_package(GSL    REQUIRED)
 find_package(Eigen3 3.3 REQUIRED NO_MODULE)
-find_package(PythonLibs REQUIRED COMPONENTS Development Interpreter Compiler)
+find_package(Python REQUIRED COMPONENTS Development)
```

Comment:
I don't think we need the Interpreter or the Compiler here. We just need the development package right?

Reply:
Correct

Reply:
Correct